examples/bits.rs

```diff
@@ -1,98 +1,207 @@
-use std::fmt;
+//! View bits in memory with turtles
+//!
+//! This example uses [@myrrlyn]'s [`bitvec`] crate to turn data into strings of
+//! bits, and then draws them on the screen.
+//!
+//! You are encouraged to change both the data used to seed the turtle, and the
+//! `bitvec` calls that control how the turtle acts, to see what changes.
+//!
+//! [@myrrlyn]: //github.com/myrrlyn
+//! [`bitvec`]: //crates.io/crates/bitvec
+
+// The `bitvec` crate provides a lot of types; we only need a few:
```

Comment:
How's

```rust
// This imports the things we need from `bitvec`, including the `Bits` trait for
// the `.bits::<_>()` method we use to view memory.
```

The explicit import is

```rust
use bitvec::prelude::{BigEndian, BitSlice, BitStore, Bits, Cursor, LittleEndian};
```

which, ew,

Reply:
The comment looks great!

Definitely not suggesting an explicit import. Even if we did that, we probably wouldn't want to go through prelude (since that's gross and not idiomatic anyway). We'd probably want to import each item from its actual module in the crate (since that helps find stuff in the docs and understand the crate structure better).

Reply:
Too bad `cargo doc` doesn't run on examples :/